recipe/meta.yaml

```diff
@@ -47,12 +47,18 @@ requirements:
     - jpeg
     - xz
     - zstd
+    - lerc
+    - libdeflate
+    - jbig
```

Comment:
Don't most of these have run exports?

Reply:
Don't know, I just replicated what was done for other dependencies like `zlib`...

Reply:
Most of them have run exports. I cleaned things up. Thanks for adding this:

https://github.com/conda-forge/libwebp-base-feedstock/blob/master/recipe/meta.yaml#L14